src/styles/colors.scss

```diff
@@ -0,0 +1,21 @@
+$black: #000000;
+$white: #FFFFFF;
+$gray: #f2f2f2;
+
+$text: $black; 
+$secondaryColor : #180C30;
+$purpleHeading: #581184;
+$lightPinkBackground: #F0EAF4; 
+$mainColor: $white;
+$deleteColor: #C91212;
```

Comment:
Create variables for specific colors or move each to above of file near color theme